geoconvert/data/countries.py

```diff
@@ -44,6 +44,7 @@
     "christmas island": "CX",  # en
     "heard island": "HM",  # en
     "norfolk island": "NF",  # en
+    "pacific island": "",  # en
```

Comment:
A similar case was encountered for "ocean indien", and the default value was `None`. I suggest to do the same here, for the data to be consistent.